tensorflow/python/client/session_ref.cc

```diff
@@ -146,7 +146,7 @@ class SessionLogger {
     // Build an index from fetch tensor name to first index in
     // output_tensor_names.
     std::unordered_map<string, int> output_name_to_offset;
-    for (int i = 0, iter_limit = output_tensor_names.size(); i < iter_limit;
+    for (int i = 0, end = output_tensor_names.size(); i < end;
          ++i) {
```

Comment:
Why keep the line broken? The header could all fit in a single line. Similar pattern in other places in the PR too